src/java.base/share/classes/java/util/random/RandomGenerator.java

```diff
@@ -496,11 +497,11 @@ default void nextBytes(byte[] bytes) {
      *
      * @return a pseudorandom {@code float} value between zero (inclusive) and one (exclusive)
      *
-     * @implSpec The default implementation uses the 24 high-order bits from a call to
-     * {@link RandomGenerator#nextInt() nextInt}().
+     * @implSpec The default implementation uses the {@link Float#PRECISION}
+     * high-order bits from a call to {@link RandomGenerator#nextInt() nextInt}().
```

Comment:
Typographical nit: please include the "()" inside the link tag for both methods.